shared/profile/search/bar.js

```diff
@@ -5,38 +5,43 @@ import * as Styles from '../../styles'
 import flags from '../../util/feature-flags'
 import Search from './container'
 
-type Props = {||}
+type Props = {|onSearch: () => void|}
 type State = {|show: boolean|}
 
 class ProfileSearch extends React.PureComponent<Props, State> {
   state = {show: false}
 
   _ref = React.createRef()
-  _onShow = () => this.setState({show: true})
+  _onShow = () => {
+    this.setState({show: true})
+    this.props.onSearch()
+  }
   _onHide = () => this.setState({show: false})
   _getAttachmentRef = () => this._ref.current
   render() {
     return (
-      <Kb.Box2 alignSelf="flex-end" direction="vertical" ref={this._ref}>
+      <Kb.Box2 style={styles.container} direction="horizontal" ref={this._ref}>
         <Kb.ClickableBox
           onClick={this._onShow}
           style={Styles.collapseStyles([styles.searchContainer, this.state.show && {opacity: 0}])}
         >
-          <Kb.Icon
-            color={Styles.globalColors.black_50}
-            fontSize={Styles.isMobile ? 20 : 16}
-            style={styles.searchIcon}
-            type="iconfont-search"
-          />
-          <Kb.Text style={styles.searchText} type="BodySemibold">
-            Search people
-          </Kb.Text>
+          <Kb.Box2 direction="horizontal">
+            <Kb.Icon
+              color={Styles.globalColors.black_50}
+              fontSize={Styles.isMobile ? 20 : 16}
+              style={styles.searchIcon}
+              type="iconfont-search"
+            />
+            <Kb.Text style={styles.searchText} type="BodySemibold">
+              Search{Styles.isMobile ? '' : ' people'}
+            </Kb.Text>
+          </Kb.Box2>
         </Kb.ClickableBox>
         <Kb.Overlay
```

Comment:
`FloatingBox` supports rendering into a `KeyboardAvoidingView`
```
  dest?: 'popup-root' | 'keyboard-avoiding-root',
```
but it's not plumbed through `Overlay`. The results list goes under the keyboard right now